### ISSUE TITLE
Vehumet needs a way to burn piety

### ISSUE THREAD

**Issue by jeremygurr** [jeremygurr/dcssca#643]
Some possibilities:

Reach (10 piety): extends any spell one time to LOS. This probably isn't that great since most high level spells go to the edge of LOS anyway.

Devastation (10 piety): allows several direct damage spells to be chosen, with one chosen target, that all get fired at the same time. 

Triple Cast (10 piety): allows one direct damage spell to be cast three times in one turn against one target. 

Fire Blossom (30 piety): every hostile target in LOS gets attacked once by a random non-area effect direct damage spell. Player doesn't need to know any spells for this to work, since they aren't chosen from what the player knows.

**Comment by Sandman25DCSS** [comment]
I like those ideas but I am not sure Vehumet needs a way to burn piety. I am a very slow player turn-wise and yet I have never dropped below 6 stars of piety once I reached them. Vanilla Vehumet does not have any abilities above 4 stars (100 piety) so I guess she is expected to always give everything she has to all characters.

**Comment by Sandman25DCSS** [comment]
So you would buff Vehumet with your changes.

**Comment by jeremygurr** [comment]
Clearly it would be a buff to vehumet. Whether Veh has room to be buffed or not I don't know. We will be able to tell if Veh is buffed too much if players almost always choosing Veh, because the advantages provided are so much greater than the alternatives. And if Veh doesn't have room to be buffed, maybe she can be nerfed in some other ways. 

It just seems wasteful to not have anything at all a player can do with their full piety. We don't have to have something to do with the excess piety, it would just avoid the player feeling like they are wasting piety, and may provide additional interesting choices. A minor issue, but one worth some consideration I think.

**Comment by Sandman25DCSS** [comment]
Yes, Vehumet is the best god for conjuration-based caster in vanilla and does not need any buffs there.

**Comment by Sandman25DCSS** [comment]
Also Veh got slightly buffed by permanent 200 piety because it maximizes chance to get MP from kills (similar thing happened to Makhleb and heal on kills).

**Comment by jeremygurr** [comment]
Mak has multiple ways to spend piety so that it's not at max all the time. But Veh doesn't. 

What if we took away mp from kills for Veh? And instead provided a high piety cost "Matter to Energy" ability that destroyed a target and converted it to mp for the caster? Maybe the piety cost could be based on the hd of the target, so very high hd characters may burn all of the player's piety. But popcorn wouldn't cost that much, but could provide valuable mp boosts.

**Comment by yrmvgh** [comment]
Vehu's in a pretty solid place, wasn't Elvy next in line for a makeover?

**Comment by Sandman25DCSS** [comment]
Trust me, Makhleb is always at max piety in my games :)

Removing MP on kills would significantly nerf Vehumet, I think it should stay because after changing Sif we don't have other gods to restore MP.

**Comment by jeremygurr** [comment]
I think I did Ely already. (see changelog)

I'm just thinking about Veh now because I'm playing a character following him. Veh feels more like a male to me (with all the destructive focus), and sif more like a female. I do like that crawl doesn't seem to make any particular gender connections here. Leaves it open to the player's imagination.

Yeah Veh doesn't _need_ anything. It's just a feeling of I wish I could do something with that piety. A player gets a feeling of loss or inefficiency, which I don't think is totally necessary here. If a player ends up with an abundance of resources, they'd like to be able to do something with it, and I think an occasional power boost that burns a slowly generating resource wouldn't hurt things too much.

**Comment by Sandman25DCSS** [comment]
> A player gets a feeling of loss or inefficiency, which I don't think is totally necessary here.

This is bad player then. Good players feel happy when they can play without using any active abilities. Chei, Makhleb, Ru, Lugonu, Ely, TSO etc. If you need god help, you are in danger.
